File: elixir/darts/lib/darts.ex
defmodule Darts do
  @type position :: {number, number}

  @doc """
  Calculate the score of a single dart hitting a target
  """
  @spec score(position) :: integer
  def score({x, y}) do
    rad = :math.sqrt(x * x + y * y)

    cond do
      rad <= 1 -> 10
      rad <= 5 -> 5
      rad <= 10 -> 1
      true -> 0
    end
  end
end

File: python/darts/darts.py
def score(x: int, y: int) -> int:
    parameter_square = x * x + y * y

    if parameter_square <= 1:
        return 10

    if parameter_square <= 25:
        return 5

    if parameter_square <= 100:
        return 1

    return 0


# def position(x: int, y: int) -> float:
#     return (x * x + y * y) ** 0.5
